### PR TITLE
Update download server page

### LINKED ISSUE #6615: [ILLUSTRATIONS UPDATE]  /download/server
@pmahnke as discussed, as we are updating illustrations and applying suru let's revamp this page a bit:

assets:

- multipass [logo](https://assets.ubuntu.com/v1/4e0ff71e-Multipass+logo_thin_rgb.svg) 320px wide

- MAAS [logo](url) width: 250px

Download Ubuntu Server   Download   Ubuntu (1)

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Add suru, logos and adjust spacing on /download/server

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/download/server
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check that the changes requested in [the issue](https://github.com/canonical-web-and-design/ubuntu.com/issues/6615) have been made

## Issue / Card

Fixes #6615

### CODE REVIEW

**Comment by anasereijo** [comment]
@steverydz thanks for doing this!

@pmahnke said he will change the Openstack section to match what we have above for MAAS and Multipass, so let's take this opportunity to do that too.

- [ ] MAAS and Multipass logos look a bit blurry to me. They are svgs so not sure what's the problem?

Screenshot 2020-02-11 at 10 28 19
Screenshot 2020-02-11 at 10 28 22

**Server section:** 

- [ ] the `hr` I used has bigger spacing than the one you used

- [ ] move the second button down to the same baseline as text

Screenshot 2020-02-11 at 10 29 30

- [ ] on mobile: fix the spacing between link and button (match what we have for the second one)
Screenshot 2020-02-11 at 10 35 25

**Comment by steverydz** [comment]
@anasereijo the source files for the maas and multipass logos look just as blurry to me

**Comment by steverydz** [comment]
@anasereijo All points have been addressed except the logos which I think look blurry when viewing them on their own too

**Comment by steverydz** [comment]
@pmahnke Ana said that you were planning to update the copy on the OpenStack section of this page. I checked the copy doc and it shows you removing this section. Should I go ahead and do that?

**Comment by steverydz** [comment]
@anasereijo this is now ready

**Comment by anasereijo** [comment]
nice! thanks

File: templates/download/server/index.html [hunk 1]
```diff
@@ -5,100 +5,64 @@
...
-      <ul class="p-list is-split">
-        <li class="p-list__item is-ticked">Supported for five years by Canonical</li>
-        <li class="p-list__item is-ticked">Updated to the OpenStack {{ releases.openstack_lts.slug }} release, including Designate, more flexible Ceph deployments and improved monitoring.</li>
-        <li class="p-list__item is-ticked">Nova LXD driver - deploy OpenStack instances as system containers, dramatically increasing per-node density</li>
-        <li class="p-list__item is-ticked">Juju OpenStack bundle -  automated deployment of OpenStack into LXD system containers</li>
-        <li class="p-list__item is-ticked">ZFS support for LXD OpenStack hosts</li>
-        <li class="p-list__item is-ticked">Certified by Microsoft to host Windows Server 2012 and Windows Server 2008 R2 as guests, under its Server Virtualisation Validation Program (SVVP)</li>
-        <li class="p-list__item is-ticked">Multi platform support OpenStack, Juju, MAAS, LXD on IBM LinuxONE mainframe, IBM Power 8 and ARM64</li>
-      </ul>
+  <div class="u-fixed-width">
```
Comment (deadlight) [inline templates/download/server/index.html]: I think that you can get a similar effect without hard-coding a style.

Something like `<div class="p-strip is-narrow"><hr></div>`

**Comment by pmahnke** [comment]
@deadlight can you re-review this please?

**Pull request merged**